ambry-protocol/src/main/java/com.github.ambry.protocol/DeleteRequest.java

```diff
@@ -26,28 +27,76 @@
  * Delete request to delete blob
  */
 public class DeleteRequest extends RequestOrResponse {
-  private BlobId blobId;
+  private final BlobId blobId;
+  private final short accountId;
+  private final short containerId;
+  private final int deletionTimeInSecs;
```

Comment:
please keep in ms (like most of the rest of the code and in line with creation time which is in ms)